Microsoft.NET.Build.Containers/ParseContainerProperties.cs

```diff
@@ -105,7 +141,7 @@ public override bool Execute()
         {
             if (!ContainerHelpers.NormalizeImageName(ContainerImageName, out string? normalizedImageName))
             {
-                Log.LogWarning(null, "CONTAINER001", null, null, 0, 0, 0, 0, $"{nameof(ContainerImageName)} was not a valid container image name, it was normalized to {normalizedImageName}");
+                Log.LogWarning(null, "CONTAINER001", "Container.InvalidImageName", null, 0, 0, 0, 0, $"{nameof(ContainerImageName)} was not a valid container image name, it was normalized to {normalizedImageName}");
```

Comment:
What's the benefit of using this overload and passing `Container.InvalidImageName`? IIRC that's the "resource name" but we've got no resources.

Reply:
I was going off of the parameter name 'helpKeyword', which made me think of some kind of f1 key or something that a user might trigger.